tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -0,0 +1,174 @@
+
+
+def generate_copyright(line_list, copyright):
+    line_list.append("<copyright>" + copyright + "</copyright>")
+
+
+def generate_tags(line_list, tags):
+    line_list.append("<tags>" + tags + "</tags>")
+
+
+def generate_icon(line_list, icon_file):
+    line_list.append("<icon>" + icon_file + "</icon>")
+
+
+def generate_license(line_list):
+    line_list.append('<license type="file">LICENSE</license>')
+
+def generate_readme(line_list):
+    line_list.append('<readme>README.md</readme>')
+
+def generate_project_url(line_list, project_url):
+    line_list.append("<projectUrl>" + project_url + "</projectUrl>")
+
+
+def generate_repo_url(line_list, repo_url, commit_id):
+    line_list.append('<repository type="git" url="' + repo_url + '"' + ' commit="' + commit_id + '" />')
+
+def generate_release_notes(line_list):
+    line_list.append("<releaseNotes>")
+    line_list.append("Release Def:")
+
+    branch = get_env_var("BUILD_SOURCEBRANCH")
+    line_list.append("\t" + "Branch: " + (branch if branch is not None else ""))
+
+    version = get_env_var("BUILD_SOURCEVERSION")
+    line_list.append("\t" + "Commit: " + (version if version is not None else ""))
+
+    line_list.append("</releaseNotes>")
+
+def generate_dependencies(xml_text, package_version, ort_package_name, ort_package_version):
+    xml_text.append("<dependencies>")
+    target_frameworks = ["NETSTANDARD" , "NETCOREAPP", "NETFRAMEWORK"]
+    for framework in target_frameworks:
+        xml_text.append(f'<group targetFramework="{framework}">')
+        xml_text.append(f'<dependency id="Microsoft.ML.OnnxRuntimeGenAI.Managed" version="{package_version}" />')
+        xml_text.append(f'<dependency id="{ort_package_name}" version="{ort_package_version}" />')
+        xml_text.append("</group>")
+
+    xml_text.append("</dependencies>")
+
+def generate_files(lines, args):
+    lines.append('<files>')
+
+    lines.append(f'<file src="{args.sources_path}\LICENSE" target="LICENSE" />')
+    lines.append(f'<file src="{args.sources_path}\README.md" target="README.md" />')
+    lines.append(f'<file src="{args.sources_path}\ThirdPartyNotices.txt" target="ThirdPartyNotices.txt" />')
+
+    def add_native_artifact_if_exists(xml_lines, runtime, artifact):
+        p = Path(f"{args.sources_path}/{args.native_build_path}/{runtime}/{args.build_config}/{artifact}")
+        if p.exists():
+            xml_lines.append(
+                f'<file src="{p.absolute()}" target="runtimes\{runtime}\\native" />'
+            )
+
+    runtimes = ["win-x64", "win-arm64", "linux-x64"]
+    for runtime in runtimes:
+      if runtime.startswith("win"):
+          add_native_artifact_if_exists(lines, runtime, "onnxruntime-genai.lib")
+          add_native_artifact_if_exists(lines, runtime, "onnxruntime-genai.dll")
+      if runtime.startswith("linux"):
+          add_native_artifact_if_exists(lines, runtime, "libonnxruntime-genai.so")
+
+    # targets
+    for dotnet in ["netstandard2.0", "net8.0", "native"]:
```

Comment:
Do we need to put the targets file in native also? I don't think we need to do that.

Reply:
This is actually from the original nuspec:

https://github.com/microsoft/onnxruntime-genai/blob/cb313b49adec51f8679701df7d2451c9d21374a2/nuget/Microsoft.ML.OnnxRuntimeGenAI.nuspec#L53

I'm no NuGet expert so I try to keep the original nuspec intact.